parm/config/config.vrfy

```diff
@@ -16,26 +16,24 @@ export CDFNL="gdas"               # Scores verification against GDAS/GFS analysi
 
 export MKPGB4PRCP="YES"           # Make 0.25-deg pgb files in ARCDIR for precip verification
 export VRFYFITS="YES"             # Fit to observations
-export VSDB_STEP1="YES"           # Populate VSDB database
-export VSDB_STEP2="NO"
-export VRFYG2OBS="YES"            # Grid to observations, see note below if turning ON
-export VRFYPRCP="YES"             # Precip threat scores
-export VRFYRAD="YES"              # Radiance data assimilation monitoring
-export VRFYOZN="YES"              # Ozone data assimilation monitoring
-export VRFYMINMON="YES"           # GSI minimization monitoring
+export VSDB_STEP1="NO"            # Disable VSDB
+export VSDB_STEP2="NO"            # Disable VSDB
+export VRFYG2OBS="NO"             # Grid to observations, see note below if turning ON
+export VRFYPRCP="NO"              # Precip threat scores
+export VRFYRAD=${VRFYRAD:-"YES"}  # Radiance data assimilation monitoring
+export VRFYOZN=${VRFYOZN:-"YES"}  # Ozone data assimilation monitoring
+export VRFYMINMON=${VRFYMINMON:-"YES"} # GSI minimization monitoring
 export VRFYTRAK="YES"             # Hurricane track verification
 export VRFYGENESIS="YES"          # Cyclone genesis verification
-export VRFYFSU="NO"               # Cyclone genesis verification (FSU)
-export RUNMOS="NO"                # whether to run entire MOS package
+export VRFYFSU="YES"               # Cyclone genesis verification (FSU)
+export RUNMOS="YES"                # whether to run entire MOS package
 
 #-------------------------------------------------
 # Fit to Observations
 #-------------------------------------------------
 
 if [ $VRFYFITS = "YES" ]; then
-
-    export fit_ver="newm.1.3"
-    export fitdir="$BASE_GIT/verif/global/Fit2Obs/${fit_ver}/batrun"
+    export fitdir="/lfs/h2/emc/global/noscrub/emc.global/git/Fit2Obs"
```

Comment:
This is a hard-wired path to WCOSS2.  Is this appropriate or a temporary measure?

Reply:
Seems it appropriate place would be in the WCOSS2 if-block, if necessary.

Reply:
It is just temporary. Will fill in when group account installed it. It will not be hardcoded path at the end.

Reply:
The directory `/lfs/h2/emc/global/noscrub/emc.global/git/Fit2Obs` does not exist.

![Screen Shot 2022-03-24 at 11 47 07 AM](https://user-images.githubusercontent.com/11394126/159955821-15c97864-ee6f-4443-a38e-a9c7568091f2.png)

`VRFYFITS` requires this utility to be installed prior to testing.  
Is it possible that this was built, installed, and tested from your personal space?
In that case, could you please include instructions to build and install to @KateFriedman-NOAA so she can install it in the `save/emc.global/git` space similar to the obsproc packages in the above screen shot.

The same question applies to `ENS_TRACKER`.  Please provide instructions to build and install that utility that typically goes under `HOMEens_tracker`.

Reply:
The path in question here is only a temp place holder.